pkg/api/addons.go

```diff
@@ -17,16 +17,24 @@ func (cs *ContainerService) setAddonsConfig(isUpdate bool) {
 	k8sComponents := K8sComponentsByVersionMap[o.OrchestratorVersion]
 	specConfig := cloudSpecConfig.KubernetesSpecConfig
 	defaultsHeapsterAddonsConfig := KubernetesAddon{
-		Name:    DefaultHeapsterAddonName,
-		Enabled: to.BoolPtr(DefaultHeapsterAddonEnabled),
+		Name:    HeapsterAddonName,
+		Enabled: to.BoolPtr(DefaultHeapsterAddonEnabled && !common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.13.0")),
```

Comment:
Carried this additional default gate over from the "last mile" "concatenate addons string" business logic area.